lib/utils/sort-locales.ts

```diff
@@ -20,7 +20,7 @@ export default function sortLocales (locales) {
   return sortByFallbackKey(localeByFallback)
 }
 
-function sortByFallbackKey (localeByFallback, key) {
+function sortByFallbackKey (localeByFallback, key?: any) {
```

Comment:
```suggestion
function sortByFallbackKey (localeByFallback, key?: string) {
```

Reply:
Should `localeByFallback` also be typed?